doc/dev/python_scheduling/interactive_scheduling_layer_norm.py

```diff
@@ -0,0 +1,190 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+# In[1]:
+
+
+import torch
+from nvfuser import (
+    FusionDefinition,
+    DataType,
+    ParallelType,
+    MemoryType,
+)
+
+
+# In[2]:
+
+
+batch_size = 1024
+tensor_size = 4096
+inputs = [
+    torch.randn(batch_size, tensor_size, dtype=torch.bfloat16, device="cuda"),
+]
+
+
+# In[3]:
+
+
+class LayerNorm(FusionDefinition):
+    def definition(self):
+        self.t0 = self.from_pytorch(inputs[0])
+        self.s0 = self.define_scalar(1e-6, dtype=DataType.Double)
+        self.norm_const = self.define_scalar(tensor_size, dtype=DataType.Int)
+
+        self.mean_cast = self.ops.cast(self.t0, dtype=DataType.Float)
+        self.sum0 = self.ops.sum(self.mean_cast, dims=[-1])
+        # NOTE Manually broadcast because fusion definition cannot access hidden reduction tensor view.
+        self.bcast_sum0 = self.ops.broadcast(self.sum0, [False, True])
```

Comment:
nitpick, doesn't look like we are using bcast_sum0 in scheduling. Does the comment above still hold?

Reply:
We don't use `bcast_sum0`  in scheduling, but we use `rfactor` on the reduction `self.sum0`.  `sum` with `keepdim=True` returns `bcast_sum0`.

Reply:
Got'ya. I didn't realize this is impacting `fn.sched.tensors())`.

This feels restrictive... i.e. should we try to lift such restrictions in the future? Would we be able to pull of an `fn.sched.all_tvs()` that can iterate through live tvs and add missing entries to fusion_state?

Reply:
Probably something to handle when I add support for concretized Fusion.